InstagramApp/InstagramApp/ActionsTableViewController.swift

```diff
@@ -0,0 +1,206 @@
+                                actionText: Constants.fourActionText,
+                                objectName: Constants.secondDogImageName,
+                                whenAction: Constants.fiveDaysAgoText)
+        
+        let five = ActionsModel(profileImageName: Constants.pizzaImageName,
+                                actionText: Constants.fiveActionText,
+                                objectName: Constants.avatarImageName,
+                                whenAction: Constants.oneDayAgoText)
+        
+        let six = ActionsModel(profileImageName: Constants.secondDogImageName,
+                               actionText: Constants.sixActionText,
+                               objectName: Constants.sunImageName,
+                               whenAction: Constants.threeDaysAgoText)
+        
+        let seven = ActionsModel(profileImageName: Constants.firstDogImageName,
+                                 actionText: Constants.fourActionText,
+                                 objectName: Constants.secondDogImageName,
+                                 whenAction: Constants.fiveDaysAgoText)
+        
+        let eight = ActionsModel(profileImageName: Constants.pizzaImageName,
+                                 actionText: Constants.fiveActionText,
+                                 objectName: Constants.avatarImageName,
+                                 whenAction: Constants.oneDayAgoText)
+        
+        let nine = ActionsModel(profileImageName: Constants.secondDogImageName,
+                                actionText: Constants.sixActionText,
+                                objectName: Constants.sunImageName,
+                                whenAction: Constants.threeDaysAgoText)
+        
+        todaysActions.append(first)
+        thisWeekActions.append(third)
+        thisWeekActions.append(four)
+        thisWeekActions.append(five)
+        thisWeekActions.append(six)
+        longTimeAgoActions.append(seven)
+        longTimeAgoActions.append(eight)
+        longTimeAgoActions.append(nine)
+    }
+
+    private func refreshPageAction() {
+        actionsRefreshControl.tintColor = .white
+        self.refreshControl = actionsRefreshControl
+        actionsRefreshControl.addTarget(self, action: #selector(endRefreshAction), for: .valueChanged)
+    }
+    
+    @objc private func endRefreshAction() {
+        let second = ActionsModel(profileImageName: Constants.pizzaImageName,
+                                  actionText: Constants.sixActionText,
+                                  objectName: Constants.firstDogImageName,
+                                  whenAction: Constants.oneMinutieAgoText)
+        todaysActions.append(second)
+        let indexPathNewRow = IndexPath(row: todaysActions.count - 1, section: 0)
+        self.tableView.insertRows(at: [indexPathNewRow], with: .automatic)
+        actionsRefreshControl.endRefreshing()
+    }
+    
+    // MARK: - UITableViewDelegate, UITableViewDataSource
```

Comment:
```suggestion
    // MARK: - UITableViewDelegate, UITableViewDataSource
```
методы делегата и Датасорс в extension